data/transactions/logic/assembler.go

```diff
@@ -1177,6 +1186,18 @@ func (ops *OpStream) Bytes() (program []byte, err error) {
 	return
 }
 
+func (ops *OpStream) addLine(err error) error {
```

Comment:
Another option for readability within this module is that ops.error(x interface{}) could type-switch on x and handle either string or error. (Kinda like log.Print())